### DISCUSSION TITLE
Hotkeys in Unit Dialog Info

### DISCUSSION THREAD

**Discussion by eos428** [ihhub/fheroes2#3440] [Ideas]
Hello everyone. Much like the previous discussion I'd like to have some more hotkeys. This time, it's for the unit dialog info :)

![Screenshot 2021-05-18 172902](https://user-images.githubusercontent.com/38433056/118636251-aef80c80-b7fe-11eb-8601-a82eabfaa71d.jpg)

We already have ESC key to exit this dialog, but let's add a few more.
- D key for dismiss
- U key for upgrade

What do you guys think?

**Comment by LeHerosInconnu** [comment]
Hello @eos428,

Even though I prefer to use the mouse, it's always nice to have another way to do it. :)

**Reply by eos428** [reply]
Yeah, I personally don't really use unit dialog info hotkeys (even in H3 or H5) but it's always nice to modernize the game a bit :D
